suites/pacific/rgw/tier-1-upgrade_4x_to_5x_latest.yaml

```diff
@@ -0,0 +1,412 @@
+      timeout: 300
+
+# Swift basic operation
+
+- test:
+    name: Swift based tests
+    desc: RGW lifecycle operations using swift
+    polarion-id: CEPH-11019
+    module: sanity_rgw.py
+    config:
+      script-name: test_swift_basic_ops.py
+      config-file-name: test_swift_basic_ops.yaml
+      timeout: 300
+
+- test:
+    name: swift versioning copy tests
+    desc: restore versioned object in swift
+    polarion-id: CEPH-10646
+    module: sanity_rgw.py
+    config:
+      script-name: test_swift_basic_ops.py
+      config-file-name: test_swift_version_copy_op.yaml
+      timeout: 500
+
+- test:
+    name: Swift based tests
+    desc: Swift bulk delete operation
+    polarion-id: CEPH-9753
+    module: sanity_rgw.py
+    config:
+      script-name: test_swift_bulk_delete.py
+      config-file-name: test_swift_bulk_delete.yaml
+      timeout: 300
+
+# resharding tests
+- test:
+    name: Resharding tests
+    desc: Reshading test - manual
+    polarion-id: CEPH-83571740
+    module: sanity_rgw.py
+    config:
+      script-name: test_dynamic_bucket_resharding.py
+      config-file-name: test_manual_resharding.yaml
+      timeout: 500
+- test:
+    name: Resharding tests
+    desc: Reshading test - dynamic
+    polarion-id: CEPH-83571740
+    module: sanity_rgw.py
+    config:
+      script-name: test_dynamic_bucket_resharding.py
+      config-file-name: test_dynamic_resharding.yaml
+      timeout: 500
+
+
+# switch from rpm to container
+- test:
+    name: switch-from-non-containerized-to-containerized-ceph-daemons
+    polarion-id: CEPH-83573510
+    module: switch_rpm_to_container.py
+    abort-on-fail: true
+
+- test:
+    name: Upgrade containerized ceph to 5.x latest
+    polarion-id: CEPH-83573679
+    module: test_ansible_upgrade.py
+    config:
+      build: '5.x'
+      ansi_config:
+        ceph_origin: distro
+        ceph_stable_release: pacific
+        ceph_repository: rhcs
+        ceph_rhcs_version: 5
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_stable: True
+        ceph_stable_rh_storage: True
+        fetch_directory: ~/fetch
+        radosgw_num_instances: 2
+        copy_admin_key: true
+        containerized_deployment: true
+        upgrade_ceph_packages: True
+        dashboard_admin_user: admin
+        dashboard_admin_password: p@ssw0rd
+        grafana_admin_user: admin
+        grafana_admin_password: p@ssw0rd
+        node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-5-dashboard-rhel8:5
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+    desc: Test Ceph-Ansible rolling update 4.x cdn -> 5.x latest -> cephadm adopt
+    abort-on-fail: True
+
+# Basic Bucket Operation Tests
```

Comment:
After the update... let's look at the cluster health and ceph versions.

Reply:
Done